### ISSUE TITLE
Game hangs at the SquareEnix logo screen.

### ISSUE THREAD

**Issue by thealtsalt** [goatcorp/FFXIVQuickLauncher#784]
### Update disclaimer

- [X] Yes, I have checked and my issue is not related to FFXIV updating and plugins not working correctly.

### What did you do?

At first I thought it was RTSS but I turned it off and the game still hangs at that screen, sometimes it will hang at the ESRB screen before it but only when im tabbed out when launching. Ive launched the game without the launcher and it doesn't hang. Also its not a permanent hang it just take a long time and even goes "the program is not responding" but if I just let it set there it will eventually progresses.

### Platform

Windows

### Relevant log output

2022-03-23 12:26:42.452 -04:00 [INF] Trying to set up Loc for language code en
2022-03-23 12:26:42.529 -04:00 [INF] XIVLauncher started as xivlauncher-6.2.20.0-6.2.20-0-g1080756
2022-03-23 12:26:42.580 -04:00 [INF] Starting update check...
2022-03-23 12:27:08.349 -04:00 [INF] Trying to set up Loc for language code en
2022-03-23 12:27:08.437 -04:00 [INF] XIVLauncher started as xivlauncher-6.2.21.0-6.2.21-0-gc98a432
2022-03-23 12:27:08.438 -04:00 [INF] Starting update check...
2022-03-23 12:27:09.813 -04:00 [INF] Engaging Autologin...
2022-03-23 12:27:09.849 -04:00 [INF] [DUPDATE] Starting...
2022-03-23 12:27:10.325 -04:00 [INF] [DUPDATE] Using release version (6.3.0.6)
2022-03-23 12:27:13.175 -04:00 [INF] [DUPDATE] Now starting for .NET Runtime 5.0.6
2022-03-23 12:27:13.518 -04:00 [INF] [DUPDATE] All set for 2022.03.01.0000.0000
2022-03-23 12:27:25.585 -04:00 [INF] LoginToGame() called
2022-03-23 12:27:25.805 -04:00 [INF] XivGame::Login(steamServiceAccount:false, cache:false)
2022-03-23 12:27:27.124 -04:00 [INF] OAuth login successful - playable:True terms:True region:2 expack:4
2022-03-23 12:27:28.334 -04:00 [INF] [HOOKS] DalamudLauncher::HoldForUpdate(gp:Q:\Games\SquareEnix\FINAL FANTASY XIV - A Realm Reborn)
2022-03-23 12:27:28.337 -04:00 [INF] XivGame::LaunchGame(steamServiceAccount:False, args:)
2022-03-23 12:27:28.338 -04:00 [INF] ArgumentBuilder::DeriveKey() rawTickCount:655800156 ticks:655800156 key:655753216
2022-03-23 12:27:28.343 -04:00 [INF] ArgumentBuilder::BuildEncrypted() checksum:"d"
2022-03-23 12:27:28.354 -04:00 [INF] [HOOKS] DalamudLauncher::Run(gp:Q:\Games\SquareEnix\FINAL FANTASY XIV - A Realm Reborn, cl:"English", pid:35796)
2022-03-23 12:27:28.355 -04:00 [INF] [HOOKS] DelayInitializeMs: 4000
2022-03-23 12:27:28.370 -04:00 [INF] [HOOKS] Started dalamud!
2022-03-23 12:27:31.529 -04:00 [INF] TROUBLESHXLTING:eyJXaGVuIjoiMjAyMi0wMy0yM1QxMjoyNzozMS41MTk2Nzc2LTA0OjAwIiwiSXNEeDExIjp0cnVlLCJJc0F1dG9Mb2dpbiI6dHJ1ZSwiSXNVaWRDYWNoZSI6ZmFsc2UsIkRhbGFtdWRFbmFibGVkIjp0cnVlLCJEYWxhbXVkTG9hZE1ldGhvZCI6MCwiRGFsYW11ZEluamVjdGlvbkRlbGF5Ijo0MDAwLjAsIlN0ZWFtSW50ZWdyYXRpb24iOmZhbHNlLCJFbmNyeXB0QXJndW1lbnRzIjp0cnVlLCJMYXVuY2hlclZlcnNpb24iOiI2LjIuMjEuMCIsIkxhdW5jaGVySGFzaCI6IjYuMi4yMS0wLWdjOThhNDMyIiwiT2ZmaWNpYWwiOnRydWUsIkRwaUF3YXJlbmVzcyI6MSwiUGxhdGZvcm0iOjAsIk9ic2VydmVkR2FtZVZlcnNpb24iOiIyMDIyLjAzLjAxLjAwMDAuMDAwMCIsIk9ic2VydmVkRXgxVmVyc2lvbiI6IjIwMjEuMTEuMjEuMDAwMC4wMDAwIiwiT2JzZXJ2ZWRFeDJWZXJzaW9uIjoiMjAyMS4xMi4xNC4wMDAwLjAwMDAiLCJPYnNlcnZlZEV4M1ZlcnNpb24iOiIyMDIxLjEyLjE0LjAwMDAuMDAwMCIsIk9ic2VydmVkRXg0VmVyc2lvbiI6IjIwMjIuMDEuMTguMDAwMC4wMDAwIiwiQmNrTWF0Y2giOnRydWUsIkluZGV4SW50ZWdyaXR5IjowfQ==

**Comment by si811** [comment]
yeah I've been having the same problem over the last few days, thought it was caused by a mod or plugin

**Comment by thealtsalt** [comment]
> yeah I've been having the same problem over the last few days, thought it was caused by a mod or plugin

Which mods do you you use with the dalamud? maybe we have some in common.



> 1.) Do you use any third party plugins? Please try removing them manually to see which of them is causing the problem: https://goatcorp.github.io/faq/dalamud_troubleshooting#q-how-do-i-manually-delete-plugins 2.) Do you use any other overlays or third party tools? Please try disabling them temporarily. 3.) If these don't help, please close this issue and contact us in our Discord server instead for direct support: https://discord.gg/3NMcUV5

ill disable able all the mods and check. The only other third party thing i use is gshade.

**Comment by si811** [comment]
These are all my mods installed through penumbra:
[DW] SHOW ME
BB_Generic_Series
Beard for Hyur Midlander
BlueUI.ttmp
Clean Replica Allagan Visor
Count_Edmonts_hair_no_bangs_for_highlander.ttmp
GamblersTrenchcoatNoStickyOuties.ttmp
Garlond Ironworks Equipment Decal Remover
hair textures 200% v2.1 update only
LatestAllagan_TF3LxFryj
Manderville - A Highlander Face 2 edit [ft. Illybites]
Manderville set - T&F3 L & XL
Omega-F Attire TF3 & Bibo+ Upscale 1.0.0
racialscaling_0_150
Scion_Healers_Corset.ttmp
Scruffy Special Highlander Edition
Shirts - T&F3 L,XL
Summer Halter & Tanga - T&F3 XL
Thavnarian Beauty Hair (Gold with Band).ttmp
Tiamat Attire Armor Set Upscales
Tight&Firm Gen3 Body - XL
Tight&Firm-Gen3Body-BreastSliders
Tight&Firm-Gen3Body-Installer
Tight&Firm-YorhaCollection-2B
Tight&Firm-YorhaCovenant

any plugins I have downloaded are through dalamud itself.

**Comment by thealtsalt** [comment]
I think the plugins are what that other person was asking about.

**Comment by thealtsalt** [comment]
The hang has sorted itself out, it was either the gpu update that i got a few days ago, or the windows update. I did them both together so i dont know which fixed it, or if they ever really did.

**Comment by wowitsmartin** [comment]
I'm getting the same issue, looks like it happens when Dalamud is injecting (disabling it fixes the issue).

I get following entries in windows event viewer:
```
Application: ffxiv_dx11.exe
CoreCLR Version: 5.0.621.22011
.NET Version: 5.0.6
Description: The process was terminated due to an internal error in the .NET Runtime at IP 00007FFADF15A037 (00007FFADEF90000) with exit code c0000005.
```

```
Faulting application name: ffxiv_dx11.exe, version: 1.0.0.0, time stamp: 0x621d1e71
Faulting module name: coreclr.dll, version: 5.0.621.22011, time stamp: 0x607f4c79
Exception code: 0xc0000005
Fault offset: 0x00000000001ca037
Faulting process ID: 0x24b8
Faulting application start time: 0x01d845d1cac18c74
Faulting application path: D:\Games\FINAL FANTASY XIV - A Realm Reborn\game\ffxiv_dx11.exe
Faulting module path: C:\Users\martin\AppData\Roaming\XIVLauncher\runtime\shared\Microsoft.NETCore.App\5.0.6\coreclr.dll
Report ID: 77ac4331-f307-4aac-ba14-7912bcf07321
Faulting package full name: 
Faulting package-relative application ID: 
```

**Comment by thealtsalt** [comment]
This issue has returned for me.

**Comment by wowitsmartin** [comment]
I ended up going back to Windows 11, no issues there - it'd seem it's win10 specific?

**Comment by thealtsalt** [comment]
> I ended up going back to Windows 11, no issues there - it'd seem it's win10 specific?

TBH it issue seems to be really intermittent and really arbitrary. Sometimes it will happen, sometimes it doesnt.

**Comment by reiichi001** [comment]
The game will momentarily hang when Dalamud is loaded and injected into FFXIV's process. It shouldn't freeze beyond 5 seconds. 

If you're clicking on the window or interacting with it in any form, Windows will then think it's not responding and assumes it must be crashing.

----
That said, it should never take long enough for Windows to make those assumptions.

Do you have mods installed through textools?

And can you please attach a recent launch's Dalamud log?

You can find the Dalamud log in `%appdata%\XIVLauncher` named `dalamud.log`. (If you only see files named `dalamud`  and `dalamud.log.old`, please give us the `dalamud` one.)
